apps/io-func-wallet-solution-backend/src/wallet-instance.ts

```diff
@@ -2,33 +2,27 @@ import * as TE from "fp-ts/TaskEither";
 import * as RTE from "fp-ts/ReaderTaskEither";
 import { pipe } from "fp-ts/function";
 
-import {
-  WalletInstanceRequest,
-  verifyWalletInstanceRequest,
-} from "./wallet-instance-request";
+import { WalletInstanceRequest } from "./wallet-instance-request";
 import { AttestationServiceConfiguration } from "./app/config";
 import { MobileAttestationService } from "./infra/attestation-service";
 
+type AttestationService = {
+  attestationServiceConfiguration: AttestationServiceConfiguration;
+};
+
 export const createWalletInstance =
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** What does this function do? Does it have side effects? In which way can it fail?‏

Reply:
Is it a question? Or are you suggesting to include the explanation in a comment?